_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 
 ### Video orientation
 
-Previously, the main video player would always play videos in full-screen regardless of your selected orientation. This was fine for landscape videos in landscape mode and portrait videos in portrait mode, but it resulted in a lot of wasted space if the video's orientation didn't match the player's. No more! There is now a full-screen button which replaces the orientation switch button, and it is _smart_. It will detect if the video is landscape or portrait, and will ensure the player is in the correct orientation when you enter full-screen! How's _that_ for convenience and usability? Not only that, but if auto-rotate is enabled, and the video is a landscape one, the button will disappear because you don't need it! (In portrait mode, you still need the button to go into full-screen from the video details.)
+In the times of old, the main video player would always play videos full-screen regardless of the user's orientation. Great if for video if held right.
+Somehow also great if held upright for perusers of vertical-video. Or, great for any user unconcerned with 90° viewing angles in either direction.
+In the glory that is NewPipe 0.20.0, a full-screen button replaces the orientation switch button. Is the video video, or some vertical nonsense? The player knows, and the player checks to see if the device is held the same way before going full-screen. If you have auto-rotate on, there is no button for regular videos, but will be to go from vertical video details, to full vertical video on the vertical. If you are into that sort of thing.
 
 <video src="/img/smart_button.webm" poster="/img/0.20.0_thumbnail_smart_fullscreen.png" class="img-responsive no-flow screenshot-image" autoplay loop muted controls></video>
 
 ### Minimized player
 
-You can now swipe down on the video in the video details page to _minimize_ it - just like YouTu... wait no. It's better. You get a minimized player at the bottom, and you can play/pause or even close it from there. How cool is that? The video will keep playing (or stay paused, if you prefer that) while you browse NewPipe at your leisure! No more having to finish a video before browsing mindlessly! Yay distractions! (In landscape mode, you can swipe down on the title to minimize as well. The swipe area is confined to that region because the rest is used for the existing volume and brightness gestures, as you know.)
+Swipe down from the video details to minimize. This gives you the cutest little video, from which you can play, pause, or close down this little rascal pulling on your heartstrings. Browse away while the video stays put, or carries on. Without distractions like these, how do you really know the video you are watching is the best possible distraction from something else? In widescreen distraction, you can swipe down on the title to minimize. Other swipage in other areas still controls volume, brightness and gestures.
```

Comment:
What does swipage mean?

Reply:
The act of swiping. I made that word up.